app/plugins/core/settings/Settings/index.js

```diff
@@ -56,6 +57,11 @@ class Settings extends Component {
           options={loadThemes()}
           onChange={value => this.changeConfig('theme', value)}
         />
+        <Text
+          label="Proxy"
+          value={proxy}
+          onChange={value => this.changeConfig('proxy', value)}
+        ></Text>
```

Comment:
change to keep default

```suggestion
        />
```

Reply:
ok, got it~